docs/content/usage/getting-started.md

```diff
@@ -44,9 +44,50 @@ option1
 : the name of the variable to which additional information should be added
 
 option2
-: supports `["value", "example", "description"]` as information scopes
+: supports `["value", "example", "description", "type", "deprecated"]` as information scopes
 
-**Example:**
+**Scoped example:**
+
+#### `value`
+
+```yaml
+# @var docker_registry_password:value: $ "secure_overwrite"
+```
+
+#### `example`
+
+```yaml
+# @var docker_registry_password:example: $ "%8gv_5GA?"
+# @var docker_registry_password:example: >
+# %8gv_5GA?
+# MyV3ryS3cur3P4ssw0rd
+# @end
+```
+
+#### `description`
+
+```yaml
+# @var docker_registry_password:description: Very secure password to login to the docker registry.
+# @var docker_registry_password:description: >
+# Multi line description are possible as well.
+# Very secure password to login to the docker registry.
+# @end
+```
+
+#### `type`
+
+```yaml
+# @var docker_registry_password:type: string
+```
+
+#### `deprecated`
+
+```yaml
+# @var docker_registry_password:deprecated: true
+# @var docker_registry_password:deprecated: since v1.0.0
+```
+
+**Full example:**
```

Comment:
Why not just continue with the full example instead of adding another format that duplicates the content?

Reply:
Adding some properties allow generating a extended array with a sum up of all good-to-know information.

Example:
| Name | Description | Type | Required | Deprecated |
|------|-------------|------|----------|------------|
| reqried_packages | Required packages for the role. | `list` | yes | no |
| reqried_packages_legacy | Required packages for the role. | `list` | yes | Yes (version > 1.0) |

Reply:
I'm happy to add some more subtypes, I'm just asking why the documentation need to be separated into `**Scoped example:**` and `**Full example:**` instead of just extending the full example.

Reply:
I faced an uncomprehension on reading the doc example. I did not understand the different capabilities in the example. That’s why I suggest to divide the example section to show possibilities and combinaisons.

Maybe scoped or full example is not useful at all.

Reply:
Fair. Let's remove the full example and just keep the scoped one? For a full example, the demo roles from https://github.com/thegeeklab/ansible-doctor/tree/main/example might be the better option anyway.

Reply:
Updated